BSB_LAN/localization/LANG_DE.h

```diff
@@ -191,6 +191,7 @@
 #define MENU_LINK_URL "https://1coderookie.github.io/BSB-LPB-LAN/kap05.html#51-url-befehle"
 #define MENU_TEXT_LOT "Telegramme loggen"
 #define MENU_TEXT_FSP "Freier Speicher auf SD Karte"
+#define MENU_TEXT_WSS "WIFI Signalstärke"
```

Comment:
This is not used in our code and might confuse translators. Please remove from the PR.

Reply:
Sorry, that was a leftover from my personal version's implementation of https://github.com/fredlcore/BSB-LAN/issues/455